src/sparseml/exporters/transforms/kv_cache/cache_keys_and_values.py

```diff
@@ -27,140 +27,143 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+OUTPUT_CACHE_NAME = """present.{attention_layer_idx}.{cache_type}"""
+INPUT_CACHE_NAME = """past_key_values.{attention_layer_idx}.{cache_type}"""
 
-# idx of key node to the key matmul
-# no great way to generically infer this from the graph since transposes can
-# be used to place it on either side of the matmul
-# hardcoding for now, will update to have a hardcoded value for each model type
-_KEY_NODE_INPUT_IDX = 0
```

Comment:
why did we take this out?

Reply:
for now it can be hard-coded as an argument to the function - no changes between CodeGen and OPT